lib/driver.js

```diff
@@ -873,10 +873,10 @@ class XCUITestDriver extends BaseDriver {
 
     let verifyProcessArgument = (processArguments) => {
       if (!_.isNil(processArguments.args) && !_.isArray(processArguments.args)) {
-        log.errorAndThrow('processArguments.args must be an array of string');
+        log.errorAndThrow('processArguments.args must be an array of strings');
       }
 
-      if (!_.isNil(processArguments.env) && !_.isObject(caps.processArguments.env)) {
+      if (!_.isNil(processArguments.env) && !_.isPlainObject(caps.processArguments.env)) {
```

Comment:
Second instance should not have `caps.`.

Reply:
fixed, good catch